scripts/gcc-cuda.cmake

```diff
@@ -25,7 +25,7 @@ set(HIOP_SPARSE ON CACHE BOOL "")
 message(STATUS "Enabling HiOp's deepchecking")
 set(HIOP_DEEPCHECKS ON CACHE BOOL "")
 
-message(STATUS "Setting default cuda architecture to 60")
-set(CMAKE_CUDA_ARCHITECTURES 60 CACHE STRING "")
+set(CMAKE_CUDA_ARCHITECTURES 60 70 75 80 CACHE STRING "")
+message(STATUS "Setting default cuda architecture to ${CMAKE_CUDA_ARCHITECTURES}")
```

Comment:
This doesn't actually seem necessary to resolve the issue. It seems like it is sufficient to just have updated Ginkgo module. Not sure if we want to keep this